index.bs

```diff
@@ -3898,9 +3901,22 @@ Note: Future variations of this specification will retain the invariant that
          will always be "<code>worker</code>" for globals implementing
          {{WorkerGlobalScope}}, and "<code>worklet</code>" for globals
          implementing {{WorkletGlobalScope}}.
-
 </div>
 
+#### The script.RealmType type ####  {#type-script-RealmType}
+
+[=Local end definition=]
+
+<pre class="cddl local-cddl">
+RealmType = {
+  type: "window" / "dedicated-worker" / "shared-worker" / "service-worker" /
+        "worker" / "paint-worklet" / "audio-worklet" / "worklet";
+}
```

Comment:
Not sure about the `type` property? Other enum values seem declared slightly differently, eg
```
LogLevel = "debug" / "info" / "warning" / "error"
```

Should this be similar?
```suggestion
RealmType = "window" / "dedicated-worker" / "shared-worker" / "service-worker" /
                       "worker" / "paint-worklet" / "audio-worklet" / "worklet"
```